README.md

```diff
@@ -1,219 +1,121 @@
-
-[![Go Report Card](https://goreportcard.com/badge/github.com/keycloak/keycloak-operator)](https://goreportcard.com/report/github.com/keycloak/keycloak-operator)
-[![Coverage Status](https://coveralls.io/repos/github/keycloak/keycloak-operator/badge.svg?branch=main)](https://coveralls.io/github/keycloak/keycloak-operator?branch=main)
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-
-# Legacy Keycloak Operator
-A Kubernetes Operator based on the Operator SDK for creating and syncing resources in Keycloak [WildFly distribution].
-
-**NOTE: This Operator is deprecated and will reach EOL with [Keycloak 20](https://www.keycloak.org/2022/03/releases).
-Consider migrating to the [new Keycloak Operator](https://github.com/keycloak/keycloak/tree/main/operator).**
+# Keycloak Realm Operator
+A Kubernetes Operator based on the Operator SDK for managing Realm and its sub-resources in Keycloak. This Operator is
+forked from the [legacy Keycloak Operator](https://github.com/keycloak/keycloak-operator) and stripped off of any functionality
+related to Keycloak Deployment as it was designed for the WildFly distribution of Keycloak and is not compatible with the
+new Quarkus distribution. The Realm Operator is meant as a **temporary workaround** to compensate for the missing
+User and Client CR in the [new Keycloak Operator](https://github.com/keycloak/keycloak/tree/main/operator) and to work
+as a side-car for it.
+
+## Motivation
+
+To provide the best experience, the new Keycloak Operator will use a new approach to manage Keycloak resources, such as Realms, Clients and Users. This approach will leverage the [new storage architecture](https://www.keycloak.org/2022/07/storage-map.html) and future immutability options, making the CRs the declarative single source of truth. In comparison to the [legacy Operator](https://github.com/keycloak/keycloak-operator), this will bring high robustness, reliability and predictability to the whole solution.
+
+Before we would consider the new Keycloak Operator ready for leveraging CRs, we expect completing several features including but not
+limited to:
+
+* File store (expected in Keycloak 20) to persist data in a file instead of DB.
+* Read-only administration REST API, UI Console and other interfaces. This is required for the new immutability concept
```

Comment:
I'd reword this to make it explicit that it's read-only for resources defined in the static store

Reply:
I added a mention of GitOps. Is it something like you had in mind?